keyboards/acheron/austin/austin.c

```diff
@@ -0,0 +1,27 @@
+#include "austin.h"
+
+void matrix_init_board(void){
+    setPinOutput(A0);
+    setPinOutput(A1);
+    setPinOutput(A2);
+}
+
+
+void led_set_kb(uint8_t usb_led) {
+    if (IS_LED_ON(usb_led, USB_LED_NUM_LOCK)) {
+        writePinHigh(A2);
+    } else {
+        writePinLow(A2);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+        writePinHigh(A0);
+    } else {
+        writePinLow(A0);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+        writePinHigh(A1);
+    } else {
+        writePinLow(A1);
+    }
+    led_set_user(usb_led);
+}
```

Comment:
Should actually be without the `!`s on the `writePin` calls, because High-On/Low-Off is the desired behaviour (with the exclamation points is Low-On/High-Off).

Reply:
This change was updated in a previouis suggestion, please see @fauxpark comment